README.md

```diff
@@ -0,0 +1,168 @@
+# Falcon
+
+<!-- TODO: -->
+<!-- markdown-link-check-disable -->
+
+[![GitHub release](https://img.shields.io/github/v/tag/Picovoice/falcon.svg)](https://github.com/Picovoice/falcon/releases)
+[![GitHub](https://img.shields.io/github/license/Picovoice/falcon)](https://github.com/Picovoice/falcon/)
+
+[![PyPI](https://img.shields.io/pypi/v/pvfalcon)](https://pypi.org/project/pvfalcon/)
+
+<!-- markdown-link-check-enable -->
+
+Made in Vancouver, Canada by [Picovoice](https://picovoice.ai)
+
+<!-- markdown-link-check-disable -->
+[![Twitter URL](https://img.shields.io/twitter/url?label=%40AiPicovoice&style=social&url=https%3A%2F%2Ftwitter.com%2FAiPicovoice)](https://twitter.com/AiPicovoice)
+<!-- markdown-link-check-enable -->
+[![YouTube Channel Views](https://img.shields.io/youtube/channel/views/UCAdi9sTCXLosG1XeqDwLx7w?label=YouTube&style=social)](https://www.youtube.com/channel/UCAdi9sTCXLosG1XeqDwLx7w)
+
+Falcon is an on-device speaker diarization engine. Falcon is:
+
+- Private; All voice processing runs locally.
+- Cross-Platform:
+    - Linux (x86_64), macOS (x86_64, arm64), Windows (x86_64)
+    - Raspberry Pi (4, 3) and NVIDIA Jetson Nano
+
+  
+## Table of Contents
+
+- [Falcon](#falcon)
+  - [Table of Contents](#table-of-contents)
+  - [AccessKey](#accesskey)
+  - [Demos](#demos)
+    - [Python Demos](#python-demos)
+    - [C Demos](#c-demos)
+  - [SDKs](#sdks)
+    - [Python](#python)
+    - [C](#c)
+  - [Releases](#releases)
+  - [FAQ](#faq)
+
+## AccessKey
+
+AccessKey is your authentication and authorization token for deploying Picovoice SDKs, including Falcon. Anyone who is
+using Picovoice needs to have a valid AccessKey. You must keep your AccessKey secret. You would need internet
+connectivity to validate your AccessKey with Picovoice license servers even though the speaker recognition is running
+100% offline.
+
+AccessKey also verifies that your usage is within the limits of your account. Everyone who signs up for
+[Picovoice Console](https://console.picovoice.ai/) receives the `Free Tier` usage rights described
+[here](https://picovoice.ai/pricing/). If you wish to increase your limits, you can purchase a subscription plan.
+
```

Comment:
Might be good to have a `What is Speaker Diarization?` section. People who really know what they are looking for will already know, but I do think it's a pretty technical term for the average user.

Reply:
Added:
```
## What is Speaker Diarization?

Speaker diarization, a fundamental step in automatic speech recognition and audio processing, focuses on identifying and
separating distinct speakers within an audio recording. Its objective is to divide the audio into segments while
precisely identifying the speakers and their respective speaking intervals.

```